src/core/config/OperationConfig.js

```diff
@@ -2261,6 +2261,20 @@ const OperationConfig = {
             }
         ]
     },
+    "From Windows Filetime To UNIX":{
+        description: "Converts a Windows Filetime timestamp to a datetime format",
+        run: DateTime.runFromFiletimeToUnix,
+        inputType: "string",
+        outputType: "string",
+        args: []
```

Comment:
Yeah, I noticed that UNIX timestamps seem to be very loosely defined outside of starting date. Windows Filetimes however have a set start time, timezone, and unit of measure.